src/components/Categories.js

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import { getCategories } from '../utils/api';
+
+function Categories({setCategory}) {
+
+    const [categories, setCategories] =useState([])
+    console.log(categories)
+    useEffect(() => {
+        getCategories().then((data) => {
+          setCategories(data.categories);
+        });
+      }, []);
+
```

Comment:
For this ticket, we need each page of categories to have its own path/url
So for example if I sent -your sites url-.com/reviews/dexterity to a friend, when they load the site it would have all the dexterity games. 
Have a think about how we could use a parametric endpoint and useParams to achieve this and please resubmit the ticket once you have